assets/queries/k8s/image_policy_webhook_not_defined/query.rego

```diff
@@ -0,0 +1,23 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.k8s as k8sLib
+
+
+CxPolicy[result] {
+	resource:= input.document[i]
+    resource.kind == "AdmissionConfiguration"
+	not hasImagePolicyWebhook(resource.plugins)
+    
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("kind={{AdmissionConfiguration}}.plugins", []),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "'ImagePolicyWebhook' should be defined in the AdmissionConfiguration plugins",
+		"keyActualValue":  "'ImagePolicyWebhook' is not defined in the AdmissionConfiguration plugins",
+	}
+}
+
```

Comment:
Maybe we should add another policy to check if there is no AdmissionConfiguration kind. What do you think?